Codespaces

```diff
@@ -0,0 +1 @@
+.devcontainer/devcontainer.json
```

Comment:
{
    // Name this configuration
    "name": "Codespace for Skills!",
    "customizations": {
        "vscode": {
            "extensions": [
                "GitHub.copilot"
            ]
        }
    }
}